distribution/docs/src/main/resources/_contents/_draft-registry-contents/managing-registry-contents.adoc

```diff
@@ -0,0 +1,348 @@
+
+|===
+
+=== Using ${ddf-registry}
+
+The *Local Registry Nodes* and *Remote Registries* tabs appear in both the ${ddf-registry} application and the ${ddf-catalog} application.
+
+==== Configuring Identity Node
+
+* Navigate to *${ddf-registry}* (or *${ddf-catalog}*) application.
+* Navigate to *Local Registry Nodes* tab.
+* Click the name of the identity node.
+* Complete all _required_ and any desired _optional_ fields.
+** Add any desired <<adding_a_service_binding,service bindings>> under the <<service_bindings, Services>> tab.
+* Click *Save*.
+** If service bindings were added, confirm setup by locating the identity node in the *Sources* tab of the *${ddf-catalog}* app.
+
+.General Information Tab
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Node Name
+|This node's name as it should appear to external systems
+|string
+|yes
+
+|Node Description
+|Short description for this node
+|string
+|yes
+
+|Node Version
+|This node's Version
+|string
+|yes
+
+|Security Attributes
+|Security attributes associated with this node.
+|String
+|
+
+|Last Updated
+|Date this entry's data was last updated
+|Date
+|
+
+|Live Date
+|Date indicating when this node went live or operational
+|Date
+|
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|===
+
+.[[service_bindings]]Services
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Service Name
+|This service name
+|string
+|
+
+|Service Description
+|Short description for this service
+|string
+|
+
+|Service Version
+|This service version
+|string
+|
+
+|Service Type
+|Identifies the type of service this is by a URN.
+|string
+|
+
+4+^|Bindings (Click *Add* to add a service binding)
+
+|Binding Name
+|This binding name
+|String
+|yes
+
+|Binding Description
+|Short description for this binding
+|String
+|
+
+|Binding Version
+|This binding version
+|
+|
+
+|Access URL
+|The url used to access this binding
+|
+|
+
+|Service Binding Type
+|The binding type for the service
+|
+|
+
+|URL Property Key
+|Property that the accessURI value should be put into for source creation
+|
+|
+
+4+|
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|===
+
+.Organizations Tab (click *Add* to add an organization)
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Organization Name
+|This organization's name
+|string
+|yes
+
+|Address
+|This organization's primary address
+|Expand to enter address information
+|yes
+
+|TelephoneNumber
+|Primary contact number for this organization
+|
+|no
+
+|Email
+|Primary contact email for this organization
+|
+|no
+
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add custom fields
```

Comment:
Same. I am wondering if it is wrong in the config and that is why you have it here